example/models.py

```diff
@@ -1,16 +1,19 @@
+from django import forms
 from django.db import models
-from wagtail.core.models import Page
-
-from wagtail_wordpress_import.models import WPImportedPageMixin
+from modelcluster.fields import ParentalManyToManyField
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
 from wagtail.core.fields import StreamField
-from wagtail.admin.edit_handlers import StreamFieldPanel
+from wagtail.core.models import Page
+from wagtail.snippets.models import register_snippet
 from wagtail_wordpress_import.blocks import WPImportStreamBlocks
+from wagtail_wordpress_import.models import WPImportedPageMixin
 
 
 class TestPage(WPImportedPageMixin, Page):
     body = StreamField(WPImportStreamBlocks)
-
+    categories = ParentalManyToManyField("example.Category", blank=True)
     content_panels = Page.content_panels + [
+        # FieldPanel("categories", widget=forms.CheckboxSelectMultiple),
```

Comment:
**Question**: Why is this commented out? It looks like it's a necessary part of the model definition.

Reply:
It's not required in the mixin i should have removed it. 

In the budgets are sexy provided wagtail app they use the promote panel for the categories and use the CheckboxSelectMultiple widget. The developer would make that decision.